experimental/packages/otlp-transformer/src/common/internal.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { SpanAttributes } from '@opentelemetry/api';
+import { fromByteArray } from 'base64-js';
+import { IAnyValue, IKeyValue } from './types';
+
+export function toAttributes(
+  attributes: SpanAttributes
+): IKeyValue[] {
+  return Object.keys(attributes).map(key => toKeyValue(key, attributes[key]));
+}
+
+export function toKeyValue(
+  key: string,
+  value: unknown
+): IKeyValue {
+  return {
+    key: key,
+    value: toAnyValue(value),
+  };
+}
+
+export function toAnyValue(value: unknown): IAnyValue {
+  if (typeof value === 'string') return { stringValue: value };
```

Comment:
nit: An optimization here on both size and perf would be to only do the typof once.
```typescript
let theType = typeof value;
if (theType === 'string')...
...
```